Node 13.13 broke babel

## Task

#### Description
TravisCI tests on node "13" and apparently 13.13 of node introduced some kind of issue with babel's imports, resulting in travis errors (local is no problem since we're not using that version of node).

```
 FAIL  src/lib/utils/__test__/index.test.js

  ● Test suite failed to run

    Cannot find module '@babel/compat-data/corejs3-shipped-proposals'

    Require stack:

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@babel/preset-env/lib/polyfills/corejs3/usage-plugin.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@babel/preset-env/lib/index.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@babel/core/lib/config/files/plugins.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@babel/core/lib/config/files/index.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@babel/core/lib/index.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/babel-jest/build/index.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@jest/transform/build/ScriptTransformer.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@jest/transform/build/index.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/jest-runtime/build/index.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@jest/core/build/cli/index.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/@jest/core/build/jest.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/jest-cli/build/cli/index.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/jest-cli/bin/jest.js

    - /home/travis/build/tvkitchen/tv-kitchen/node_modules/jest/bin/jest.js
```

Lets invoke the fix of updating deep dependencies.

#### Relevant Resources / Research
* The [babel issue](https://github.com/babel/babel/issues/11439) which provides a solution.

#### Related Issues
A list of related issues

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "@babel/plugin-proposal-class-properties": "^7.8.3",
     "@babel/preset-env": "^7.8.7",
     "@babel/register": "^7.8.6",
-    "babel-eslint": "^10.1.0",
+    "babel-eslint": "^8.2.6",
     "eslint": "^6.8.0",
     "eslint-config-airbnb-base": "^14.1.0",
     "eslint-import-resolver-alias": "^1.1.2",

